open-reac/src/main/java/com/powsybl/openreac/parameters/input/VoltageLevelLimitsOverrideInput.java

```diff
@@ -32,62 +32,70 @@ public class VoltageLevelLimitsOverrideInput implements AmplInputFile {
-                double value = (voltageLimitOverride.isRelative() ? newLimits.getFirst() : 0.0) + voltageLimitOverride.getLimit() / nominalV;
-                newLimits.setFirst(value);
-            } else if (voltageLimitOverride.getVoltageLimitType() == VoltageLimitOverride.VoltageLimitType.HIGH_VOLTAGE_LIMIT) {
-                double value = (voltageLimitOverride.isRelative() ? newLimits.getSecond() : 0.0) + voltageLimitOverride.getLimit() / nominalV;
-                newLimits.setSecond(value);
-            } else {
-                throw new UnsupportedOperationException("Unsupported voltage limit type: " + voltageLimitOverride.getVoltageLimitType());
+            for (VoltageLimitOverride voltageLimitOverride : entry.getValue()) {
+                if (voltageLimitOverride.getVoltageLimitType() == VoltageLimitOverride.VoltageLimitType.LOW_VOLTAGE_LIMIT) {
+                    double value = (voltageLimitOverride.isRelative() ? newLimits.getFirst() : 0.0) + voltageLimitOverride.getLimit() / nominalV;
+                    newLimits.setFirst(value);
+                } else if (voltageLimitOverride.getVoltageLimitType() == VoltageLimitOverride.VoltageLimitType.HIGH_VOLTAGE_LIMIT) {
+                    double value = (voltageLimitOverride.isRelative() ? newLimits.getSecond() : 0.0) + voltageLimitOverride.getLimit() / nominalV;
+                    newLimits.setSecond(value);
+                } else {
+                    throw new UnsupportedOperationException("Unsupported voltage limit type: " + voltageLimitOverride.getVoltageLimitType());
+                }
             }
-
             if (newLimits.getFirst() >= newLimits.getSecond()) {
                 throw new InvalidParametersException(OVERRIDE_ON_VOLTAGE_LEVEL + voltageLevelId + " leads to low voltage limit >= high voltage limit.");
             }
-            if (newLimits.getFirst() < 0.5) {
-                LOGGER.warn("Voltage level {} has a low voltage limit lower than 0.5 PU ({} PU)", voltageLevelId, newLimits.getFirst());
-            }
-            if (newLimits.getSecond() > 1.5) {
-                LOGGER.warn("Voltage level {} has a high voltage limit greater than 1.5 PU ({} PU)", voltageLevelId, newLimits.getSecond());
-            }
             normalizedVoltageLimitsOverride.put(voltageLevelId, newLimits);
         }
-
-        for (Map.Entry<String, Pair<Double, Double>> entry : normalizedVoltageLimitsOverride.entrySet()) {
-            String voltageLevelId = entry.getKey();
-            if (entry.getValue().getFirst() > 1.) {
-                throw new InvalidParametersException(OVERRIDE_ON_VOLTAGE_LEVEL + voltageLevelId + " leads to low voltage limit > nominal voltage.");
-            }
-            if (entry.getValue().getSecond() < 1.) {
-                throw new InvalidParametersException(OVERRIDE_ON_VOLTAGE_LEVEL + voltageLevelId + " leads to high voltage limit < nominal voltage.");
-            }
-        }
```

Comment:
Is it OK to remove this control?
(This change was not reported in the PR description.)

Reply:
Indeed, the change was not reported in the PR description. It was done in another PR (#99), whose commits were included in this PR, for test purposes. I have to admit that it is not clear.

Reply:
OK, it's alright then :+1: